app/javascript/src/Stats/buyer/stats_application.jsx

```diff
@@ -43,13 +43,13 @@ export class StatsApplicationChartManager extends StatsUsageChartManager {
   }
 
   _metricUrl (id) {
-    return `/stats/applications/${id}/summary.json?version=2.0`
+    return `/stats/api/applications/${id}/summary.json?version=2.0`
```

Comment:
So this is a temporary measure. we would need to clean that by properly announcing a deprecation notice
On the API side

`stats/applications` => `admin/api/stats/applications`

I wonder if we could create the routes already